File: files/100-199/115/CH-115.py
import pandas as pd

path = "CH-115 Multi Replacement.xlsx"
input = pd.read_excel(path, usecols="B:E", skiprows=1)
test = pd.read_excel(path, usecols="G:J", skiprows=1).rename(columns=lambda x: x.replace(".1", ""))

result = input.replace({"0{3}|q": "-"}, regex=True)

print(result.equals(test))  # True

File: files/100-199/115/CH-115.R
library(tidyverse)
library(readxl)

path = "files/CH-115 Multi Replacement.xlsx"
input = read_excel(path, range = "B2:E9")
test  = read_excel(path, range = "G2:J9")

result = input %>%
  mutate(across(c(`Product ID`, `Customer ID`), ~str_replace(., "0{3}|q", "-")))

identical(result, test)
#> [1] TRUE